### PR TITLE
{tools}[gompi/2022a] IOR v3.3.0

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/a717f8060e6ebef04f8459c0b4bfbebf for a full test report.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/i/IOR/IOR-3.3.0-gompi-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Åke Sandgren, HPC2N, Umeå University
```
Comment (lexming) [inline easybuild/easyconfigs/i/IOR/IOR-3.3.0-gompi-2022a.eb:2]: Please keep the attributions of previous easyconfigs.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @akesandgren !

**Pull request merged**